cmd/repo-updater/repoupdater/server.go

```diff
@@ -27,9 +27,13 @@ import (
 type Server struct {
 	repos.Store
 	*repos.Syncer
-	GithubDotComSource interface {
+	SourcegraphDotComMode bool
+	GithubDotComSource    interface {
```

Comment:
nit:

```suggestion
	GitHubDotComSource    interface {
```

If causes too many cascading changes... then nvm.

Reply:
I'd rather do a separate PR to address this casing inconsistency everywhere.